components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -566,6 +566,7 @@ public final class APIConstants {
     public static final String DEFAULT_ORGANIZATION_CLAIM_NAME = "http://wso2.org/claims/organization";
     public static final String DEFAULT_TOKEN_TYPE = "DEFAULT";
     public static final String TOKEN_TYPE_OAUTH = "OAUTH";
+    public static final String DEFAULT_TOKEN_TYPE_NEW = "Default";
```

Comment:
Shall we add a meaningful name

Reply:
Added in https://github.com/wso2/carbon-apimgt/pull/11497/commits/daa309d5b1703883b0810165a12fc117b451ce28